courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb

```diff
@@ -0,0 +1,937 @@
+{
```

Comment:
<p>Would suggest:</p><p> </p><p><strong>Lab Scope</strong>:  This lab does not cover how to make predictions on the model or deploy it to Cloud AI Platform.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
thanks!